figs/stanplots/analyses_wanang.R

```diff
@@ -34,9 +46,121 @@ l1W_ranef <- brm(log(BIO) ~ TREAT + (1+TREAT|GARDEN),
            control = list(adapt_delta = 0.99),
            file="bioAllrenef")
 
-# Summaries
-summary(l1W)
-summary(l1W_ranef)
+
+# random intercepts and slopes --------------------------------------------
+
+#define model 
+#does changing contrasts change the model?
+levels(AllTestData$TREAT)
+#contrasts(AllTestData$TREAT) <- contr.helmert(n = 6)
+
+bio_lnorm_bf <- bf(BIO ~ 1 + TREAT + (1 + TREAT|GARDEN), family = lognormal())
+brms::get_prior(bio_lnorm_bf, data = AllTestData)
+## Set some priors 
+priors <- c(set_prior("normal(4, 2)", class = "Intercept"),
+            set_prior("normal(0, 3)", class = "b"),
+            set_prior("normal(0, 4)", class = "sd"),
+            set_prior("normal(0, 3)", class = "sigma"),
+            set_prior("lkj(2)", class = "cor"))
+
+## Whole community with random intercept and slope
+l1W_ranef <- brm(bio_lnorm_bf, 
+                 data=AllTestData,
+                 prior = priors,
+                 control = list(adapt_delta = 0.95),
+                 file="bioAllrenef_lnormal", cores = 4, sample_prior = TRUE)
+
+stancode(l1W_ranef)
+
+#what are the parameters here worth thinking about?
+brms::parnames(l1W_ranef)
+#get the info on divergent iterations
+np <- nuts_params(l1W_ranef)
+# customize appearance of divergences
+color_scheme_set("darkgray")
+div_style <- parcoord_style_np(div_color = "orange", div_size = 1, div_alpha = 1)
+mcmc_parcoord(as.array(l1W_ranef),
+              size = 0.25, alpha = 0.1,
+              regex_pars = "sd_.*",
+              np = np, np_style = div_style)+ coord_flip()
```

Comment:
This is a technique for looking at those "divergent iterations" which is supposed to help you.. think about your model? its not straightforward

Reply:
hmmm